gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -199,11 +199,6 @@ def _get_invalid_file_contents(self):
             '#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	Sample\n',
             '1    1  '
         ],
-        # Missing "GT:GQ" format, but GQ is provided.
```

Comment:
Why is it removed? You mentioned this makes test fail. But do you know why? How is it related to cython?

Reply:
This test is failing when pyvcf uses cython, this is a known issue:
https://github.com/apache/beam/pull/4221

We have to remove it otherwise test does not pass.

Reply:
Ah, this is really bad. 
On the plus side, C PyVcf is more robust to malformed input. :-)